ci/ci/github.py

```diff
@@ -486,27 +486,37 @@ async def _update_last_known_github_status(self, gh):
                 self.target_branch.state_changed = True
 
     async def _update_github_review_state(self, gh):
-        latest_state_by_login = {}
-        async for review in gh.getiter(
-            f'/repos/{self.target_branch.branch.repo.short_str()}/pulls/{self.number}/reviews'
-        ):
-            login = review['user']['login']
-            state = review['state']
-            # reviews is chronological, so later ones are newer statuses
-            if state != 'COMMENTED':
-                latest_state_by_login[login] = state
-
-        review_state = 'pending'
-        for login, state in latest_state_by_login.items():
-            if state == 'CHANGES_REQUESTED':
-                review_state = 'changes_requested'
-                break
-            if state == 'APPROVED':
-                review_state = 'approved'
-            else:
-                assert state in ('DISMISSED', 'COMMENTED', 'PENDING'), state
+        review_state_query = f"""
+        query {{
+            repository(owner: "{self.target_branch.branch.repo.owner}", name: "{self.target_branch.branch.repo.name}") {{
+                pullRequest(number: {self.number}) {{
+                      number
```

Comment:
Are you using the number?

Reply:
Ooh, good catch. I was using this as a sanity check when testing the API out manually. It's not really necessary any more, assuming we trust github to return what we ask it for